housingbayarea/bloom
# fix: update resources

# Pull Request Template

## Issue Overview

This PR addresses #3457


## Description

updates in resources from Alameda described in task.

## How Can This Be Tested/Reviewed?

having env set to Alameda go to `additional-resources`. Check updates from task for each language.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
